Failed to load the git-tabs package

Upon installation, I get an error message, stating: `Cannot find module 'fs-plus'`.

**Atom Version**: 1.2.4
**System**: Fedora 22
**Thrown From**: [git-tabs](https://github.com/anderoonies/git-tabs) package, v0.2.0


### Stack Trace

Failed to load the git-tabs package

```
At Cannot find module 'fs-plus'

Error: Cannot find module 'fs-plus'
    at Module._resolveFilename (module.js:336:15)
    at Function.Module._resolveFilename (/usr/share/atom/resources/app.asar/src/module-cache.js:383:52)
    at Function.Module._load (module.js:286:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/home/oli/.atom/packages/git-tabs/lib/git-tabs.coffee:2:6)
    at Object.<anonymous> (/home/oli/.atom/packages/git-tabs/lib/git-tabs.coffee:1:1)
    at Module._compile (module.js:434:26)
    at Object.defineProperty.value [as .coffee] (/usr/share/atom/resources/app.asar/src/compile-cache.js:190:21)
    at Module.load (module.js:355:32)
```

### Commands

```
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "wrap-guide",
      "vim-mode",
      "smart-tab-name",
      "rename-tabs",
      "multirow-tabs"
    ]
  }
}
```

### Installed Packages

```coffee
# User
color-tabs, v0.1.8
foldername-tabs, v0.1.5
git-tabs, v0.2.0
linter, v1.11.3
linter-flake8, v1.9.2
minimap, v4.16.2
multi-wrap-guide, v0.23.2
pigments, v0.19.1
tidy-tabs, v0.3.0

# Dev
No dev packages
```

--- a/package.json
+++ b/package.json
@@ -10,10 +10,11 @@
   "engines": {
     "atom": ">=1.0.0 <2.0.0"
   },
-  "dependencies": {},
-  "devDependencies": {
-    "coffeelint": "^1.11.1",
+  "dependencies": {
     "fs-plus": "^2.8.0"
+  },
+  "devDependencies": {
+    "coffeelint": "^1.11.1"
   },
   "scripts": {}
 }

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "git-tabs",
   "main": "./lib/git-tabs",
-  "version": "0.2.0",
+  "version": "0.3.0",
   "description": "Preserve your tabs on a per-branch and per-repo basis",
   "keywords": [],
   "activationCommands": {},

